{"n_dim": 55, "n_trials": 550, "objective": "markowitz_return_on_cube", "white": "scipy_powell", "black": "dlib_default", "traceback": ["passing", "Traceback (most recent call last):\n  File \"/opt/hostedtoolcache/Python/3.8.16/x64/lib/python3.8/site-packages/humpday/comparison/eloratings.py\", line 76, in optimizer_game\n    black_best_val, black_best_x, black_feval_count = black(objective, n_trials=n_black_trials,\n  File \"/opt/hostedtoolcache/Python/3.8.16/x64/lib/python3.8/site-packages/humpday/optimizers/dlibcube.py\", line 27, in dlib_default_cube\n    best_x, best_val = find_min_global(_objective, lb, ub, n_trials)\nRuntimeError: \n\nError detected at line 63.\nError detected in file /tmp/pip-install-dawomi94/dlib_14fffb8721e845fc862ea167899872ae/tools/python/src/global_optimization.cpp.\nError detected in function double call_func(pybind11::object, const dlib::matrix<double, 0, 1>&).\n\nFailing expression was 0 < num && num <= 35.\nFunctions being optimized must take between 1 and 35 scalar arguments.\n\n"], "best_val": [1068.900367356087, null], "best_x": [[6.610696135189609e-05, 6.610696135189609e-05, 6.610696135189609e-05, 6.610696135189609e-05, 0.0002467002445856805, 6.610696135189609e-05, 0.0002461728487229557, 6.610696135189609e-05, 7.362997544048794e-05, 6.610696135189609e-05, 6.610696135189609e-05, 0.00020640360761261994, 0.0001397369358118593, 7.362997544048794e-05, 0.0001397369358118593, 7.362997544048794e-05, 6.610696135189609e-05, 0.00017307027171223962, 0.0002800335820725831, 0.00020062746965100004, 9.111140454920059e-05, 7.362997544048794e-05, 6.610696135189609e-05, 0.0001973413478252191, 0.00019992647641844377, 6.610696135189609e-05, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0], null], "feval_count": [551, null], "n_trials_instructed": [550, 551], "passing": [true, false], "completed": false, "winner": "incomplete", "points": null}